tests/function/test_config.py

```diff
@@ -614,3 +615,272 @@ def test_interval_in_virtwho_sysocnfig(self, virtwho, function_sysconfig):
                 and result['loop'] == 60)
 
         function_sysconfig.clean()
+
+    @pytest.mark.tier1
+    def test_pid_files_permission(self, virtwho, ssh_host):
+        """Test the sysconfig and pid files permission
+
+        :title: virt-who: config: test the sysconfig and pid files permission
+        :id: 21bf00ef-8b58-48b8-a36c-d65bc7b18dc0
+        :caseimportance: High
+        :tags: tier1
+        :customerscenario: false
+        :upstream: no
+        :steps:
+
+            1. Stop virt-who to check virt-who sysconfig permission
+            2. Start virt-who to check virt-who.pid file permission
+
+        :expectedresults:
+
+            1. the permission of /etc/sysoncifg/virt-who file should be -rw-------
+            2. the permission of /var/run/virt-who.pid file should be  -rw-------
+        """
+        # check virt-who sysconfig file permission
+        virtwho.stop()
+        cmd = f"ls -l '{SYSCONFIG_FILE}'"
+        ret, output = ssh_host.runcmd(cmd)
+        assert (ret == 0
+                and output is not None
+                and output != ""
+                and "-rw-------" in output)
+
+        # check virt-who.pid file permission
+        pid_file = "/var/run/virt-who.pid"
+        virtwho.start()
+        cmd = f"ls -l '{pid_file}'"
+        ret, output = ssh_host.runcmd(cmd)
+        assert (ret == 0
+                and output is not None
+                and output != ""
+                and "-rw-------" in output)
+
+@pytest.mark.usefixtures('function_globalconf_clean')
+@pytest.mark.usefixtures('class_hypervisor')
+@pytest.mark.usefixtures('class_virtwho_d_conf_clean')
+class TestConfigurationNegative:
```

Comment:
let's make it uniform to add the `positive` and `negative` keyword, my suggestion is
`:title: virt-who: config: test debug option (positive)`
`:title: virt-who: config: test debug option (negative)`